tsl/test/expected/continuous_aggs_invalidation.out

```diff
@@ -207,8 +253,7 @@ SELECT hypertable_id AS hyper_id,
  hyper_id | start | end 
 ----------+-------+-----
         2 |    20 |  20
-        2 |    30 |  80
-(2 rows)
+(1 row)
```

Comment:
The  `continuous_aggs_materialization_invalidation_log` is shown in the next query below this one.

The hypertable invalidation log contains invalidations for _all_ hypertables, in this case ID 1 and 2. During a refresh, only the entries that are on hypertable 1, which the cagg exists on, will be moved over. Refreshing a cagg on the other hypertable would move those entries over.

There's a comment above that makes this clear.

But this PR is about moving the invalidation threshold, so moving invalidations between logs is not the subject of this PR. The only reason there's a change here is because now it was possible to replace the `REFRESH MATERIALIZED VIEW` command with a `continuous_aggregate_refresh` on a window (since it now moves the invalidation threshold so that new invalidation are generated).

Reply:
Thanks for the clarification. Yes, I see the logs for the materialization_invalidation_log below this test case.  I thought the invalidations belonged to hypertable-id 2 and not to 1.